site/blog/2016/jul/1/sphinx-and-rtd-for-writers.rst

```diff
@@ -0,0 +1,552 @@
+
+Treating documentation as code is becoming a major theme in the software industry.
+This is coming from both sides,
+with developers starting treat documentation as a priority alongside tests and code,
+and writers seeing a lot of value in integrating more into the development process.
+This marrying of cultures isn't simple,
+but having the proper tools for the job makes both sides happy with the results that get produced.
+
+A lot of developer tools don't work well for writers.
+Sphinx and Read the Docs are unique in this ecosystem,
+in that they have powerful tools that both writers and developers want,
+in one convenient package.
+
+Overview of the Ecosystem
+-------------------------
+
+Read the Docs is the largest open source documentation hosting site in the world.
+Open Source in this context means both that the code is open source,
+and the documentation hosted is for open source software.
+It is a developer focused platform,
+but it has most of the features that technical writers have come to expect in a tool as well.
+This blending of worlds makes it the best suited platform for teams that want both writers and developers contributing to product and API documentation.
+
+Read the Docs is best thought of as a continuous documentation platform for Sphinx.
+Continuous documentation is analogous to continuous integration of code.
+In practice it means that your documentation is built,
+tested, 
+and updated on every commit.
+
+Sphinx provides a documentation generator that is best-in-class for software docs.
+Sphinx documents are written in the reStructuredText markup language.
+which is similar to Markdown,
+but much more powerful.
+Read the Docs provides a hosting platform for Sphinx.
+
+As a writer who uses Sphinx,
+your day to day is writing reStructuredText in plain text files.
+You then build your documentation by running Sphinx on the command line.
+Generally it's easiest to output HTML for local writing and testing,
+and then you can let Read the Docs generate PDF's and other formats.
+
+This article provides an overview of the features of Sphinx and Read the Docs,
+and allow you to evaluate them for use in your organization.
+
+Introduction to Sphinx
+----------------------
+
+`Sphinx <http://www.sphinx-doc.org/>`_ is the documentation tool of choice for the Python language community,
+and increasingly also for other programming languages and tools.
+It was `originally created <https://en.wikipedia.org/wiki/Sphinx_(documentation_generator)>`_ in 2008 to document the Python language itself.
+
+Over the past eight years,
+it has turned into a robust and mature solution for software documentation.
+It includes a number of features that writers expect,
+such as:
+
+* Single Source Publishing 
+    * Output HTML, PDF, ePub, and more
+    * Content reuse through includes
+    * Conditional includes based on content type and tags
+* Multiple mature HTML themes that look beautiful on mobile and desktop
```

Comment:
`look beautiful` isn't compelling, this could rather describe how documentation is suddenly highly usable on multiple devices or something a little more explicit